tests/bearlib/languages/LanguageDefinitionTest.py

```diff
@@ -13,16 +13,6 @@ def setUp(self):
         self.section = Section("any")
         self.section.append(Setting("language", "CPP"))
 
-    def test_nonexistant_file(self):
-        self.section.append(Setting("language", "bullshit"))
-
-        with self.assertRaises(FileNotFoundError):
-            LanguageDefinition.from_section(self.section)
```

Comment:
well I added the doctest to cover our new code so it made sense to remove the redundant tests here